copilot/README.md

```diff
@@ -0,0 +1,93 @@
+# Deploying on AWS
+
+## Introduction
+
+[Copilot](https://aws.github.io/copilot-cli/) helps simplify AWS resources and
+automate deploymnents for projects.
+
+This sample configuration runs the Open Assistant web app as an ECS Fargate
+services backed by a Serverless Aurora Postgres database.
+
+## To Setup
+
+Setup requires a few steps:
+
+```sh
+copilot app init --domain your_domain.tls
```

Comment:
is the TLS here to indicate a certificate? or should it be TLD (top-level domain)?

Reply:
It's for certificates.  I expanded the comment to be more clear.

Reply:
Re the signup page changes, I needed to change the Discord code to make sure it works end to end after a prior PR disabled it.  I added in the GItHub one since it was straightforward.